src/codeu/chat/server/View.java

```diff
@@ -89,6 +89,9 @@ public View(Model model) {
     return all;
   }
 
+  public ServerInfo getInfo() {
+    return null;
```

Comment:
This isn't quite right - check out the top of page 7 of the guide. In general, a function that only returns null in java means something isn't right.